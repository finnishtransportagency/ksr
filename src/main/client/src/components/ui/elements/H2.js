import styled from 'styled-components';

export const H2 = styled.h2`
    font-weight: 500;
    font-size: 18px;
    text-transform: capitalize;
`;
