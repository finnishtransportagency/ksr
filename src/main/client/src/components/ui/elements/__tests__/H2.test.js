import { shallow } from 'enzyme';
import 'jest-styled-components';
import React from 'react';
import { H2 } from '../H2';

describe('<H2 />', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<H2 />);
        expect(wrapper).toMatchSnapshot();
        expect(wrapper).toHaveStyleRule('font-size', '18px');
        expect(wrapper).toHaveStyleRule('font-weight', '700');
    });
});
