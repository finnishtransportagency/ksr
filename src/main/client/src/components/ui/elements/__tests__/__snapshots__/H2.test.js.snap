// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<H2 /> renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <styled.h2 />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "className": "sc-bdVaJa gYoROP",
    },
    "ref": null,
    "rendered": null,
    "type": "h2",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "sc-bdVaJa gYoROP",
      },
      "ref": null,
      "rendered": null,
      "type": "h2",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
